abci/README.md

```diff
@@ -29,7 +29,7 @@ The two guides to focus on are the `Application Development Guide` and `Using AB
 To compile the protobuf file, run:
 
 ```
-make protoc
```

Comment:
To recompile the proto files one currently still has to delete `abci/types/types.pb.go` or not? `make protoc_abci` doesn't have any effect, if the file already exists?

Reply:
no, if there're some changes, it will apply them